scripts/dev/dockerfile_generator.py

```diff
@@ -2,40 +2,53 @@
 import argparse
 
 
-def operator_params():
+def operator_params(adds):
```

Comment:
I think this name could be improved, I didn't know what this was until I saw the description + Dockerfile template, how about `files_to_add`? (also, type hinting!)